apps/home/templates/authenticated-home.html

```diff
@@ -31,6 +32,18 @@
     <div class="ds-l-lg-col--11 ds-l-md-col--11 ds-l-sm-col--12">
       {% include "include/messages.html" %}
 
+
+      {% switch message_implicit_flow %}
+        <div class="ds-c-alert ds-c-alert--warn ds-u-margin-bottom--4">
+          <div class="ds-c-alert__body">
+            <h3 class="ds-c-alert__heading">Announcement</h3>
+              <p class="ds-c-alert__text">
+                <a href="javascript:void(0);">The Implicit Auth Flow is being deprecated and should not be used for development.</a>
```

Comment:
Looks like the link is still need, correct?

`<a href="https://groups.google.com/g/developer-group-"
                                                 "for-cms-blue-button-api/c/PFfVNymltfE/m/XsRMaQXXCAAJ">The Implicit Auth Flow is being deprecated and should not be used for development.</a>`

Reply:
TY for catching that!